ansible/roles/pmm2-images/tasks/main.yml

```diff
@@ -0,0 +1,249 @@
+---
+    - name: Packages                   | Add Nginx repository
+      yum_repository:
+        name: nginx
+        description: nginx repo
+        baseurl: http://nginx.org/packages/mainline/centos/7/$basearch/
+        gpgcheck: no
+        enabled: yes
+
+    - name: Packages                   | Add PMM2 Server YUM repository
+      yum_repository:
+        name: pmm2-laboratory
+        description: PMM Server YUM repository - x86_64
+        baseurl: https://repo.percona.com/pmm2-components/yum/{{ pmm2_server_repo }}/7/RPMS/x86_64/
+        gpgcheck: yes
+        enabled: no
+        gpgkey: file:///etc/pki/rpm-gpg/PERCONA-PACKAGING-KEY
+
+    - name: Packages                   | Update OS
+      yum:
+        name: '*'
+        state: latest
+        exclude: 'ansible'
+        disablerepo: percona-release-x86_64
+
+    - name: Packages                   | Install OS tools
+      yum:
+        name:
+          - screen
+          - yum-utils
+          - yum-cron
+          - logrotate
+          - unzip
+          - python2-pip
+
+    - name: Install supervisord
+      include_role:
+        name: supervisord-init
+
+    - name: Create dirs                | Create dirs
+      file: path={{ item }} state=directory
+      with_items:
+        - /var/lib/cloud/scripts/per-once
+        - /var/lib/cloud/scripts/per-boot
+
+    - name: Configure yum              | Configure yum-cron
+      replace:
+        dest: /etc/yum/yum-cron-hourly.conf
+        regexp: 'update_cmd = default'
+        replace: 'update_cmd = minimal-security'
+
+    - name: Configure yum              | Configure yum-cron
+      replace:
+        dest: /etc/yum/yum-cron.conf
+        regexp: 'update_cmd = default'
+        replace: 'update_cmd = minimal-security'
+
+    - name: Install RPMs               | Install RPMs for PMM2 server
+      yum:
+        name:
+          - nginx-1.15.8
```

Comment:
I think we can bump it.

Reply:
I suggest doing it via a separate task/review